app/models/katello/erratum.rb

```diff
@@ -3,8 +3,8 @@ class Erratum < Katello::Model
     include Concerns::PulpDatabaseUnit
 
     SECURITY = "security".freeze
-    BUGZILLA = "bugfix".freeze
-    ENHANCEMENT = "enhancement".freeze
+    BUGZILLA = ["bugfix", "recommended"].freeze
+    ENHANCEMENT = ["enhancement", "optional"].freeze
```

Comment:
For simplicity, maybe we should put all the types in an array?

Reply:
sure